org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/binding/wizard/connection/UsernamePasswordWizardPage.java

```diff
@@ -72,6 +82,23 @@ protected void doCreateControl(Composite container) {
 
     WizardPageSupport.create(this, dataBindingContext);
   }
+  
+  private static void createDeprecationLabel(final Composite container) {
+    var deprecationContainer = new Composite(container, SWT.NONE);
+    deprecationContainer.setLayout(new GridLayout(3, false));
+    deprecationContainer.setLayoutData(new GridData(SWT.LEFT, SWT.DOWN, true, false, Integer.MAX_VALUE, 1));
+    
+    var labelIcon = new Label(deprecationContainer, SWT.NULL);
+    labelIcon.setImage(SonarLintImages.IMG_SEVERITY_BLOCKER);
+    
+    var labelDeprecation = new Label(deprecationContainer, SWT.NULL);
+    labelDeprecation.setText(DEPRECATION_MESSAGE);
+    labelDeprecation.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));
+    labelDeprecation.setFont(JFaceResources.getFontRegistry().getBold(JFaceResources.DEFAULT_FONT));
+    
+    labelIcon = new Label(deprecationContainer, SWT.NULL);
```

Comment:
Maybe use different variables to clear confusion? `labelLeft` and `labelRight` or something similar.